restapi/src/main/java/io/stargate/web/swagger/SwaggerUIResource.java

```diff
@@ -90,7 +92,7 @@ public Response get(
       token =
           Strings.isNullOrEmpty(bearerToken)
               ? bearerToken
-              : bearerToken.replaceFirst("^Bearer\\s", "");
+              : bearerTokenPattern.matcher(bearerToken).replaceFirst("");
```

Comment:
Minor performance optimization since it's true regexp and seems like it might get called quite regularly.